SPECS/ocaml-sexplib/ocaml-sexplib.spec

```diff
@@ -0,0 +1,337 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+%ifnarch %{ocaml_native_compiler}
+%global debug_package %{nil}
+%endif
+
+%bcond_with vim
+
+%global srcname sexplib
+
+Name:           ocaml-%{srcname}
+Version:        0.16.0
+Release:        1%{?dist}
+Summary:        Automated S-expression conversion
+
+# The project as a whole is MIT, but code in the src subdirectory is BSD.
+License:        MIT and BSD
+URL:            https://github.com/janestreet/%{srcname}
+Source0:        %{url}/archive/v%{version}/%{srcname}-%{version}.tar.gz
+
+BuildRequires:  ocaml >= 5.1.1
+BuildRequires:  ocaml-dune >= 2.0.0
+BuildRequires:  ocaml-num-devel
+BuildRequires:  ocaml-parsexp-devel >= 0.16
+BuildRequires:  ocaml-sexplib0-devel >= 0.16
+BuildRequires:  vim
+
+%if %{with test}
```

Comment:
Please use the `with_check` macro for test-only dependencies. Right now they are always going to be ignored.

Reply:
changed it